scvi/dataset/cite_seq.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+import pandas as pd
+
+from .dataset import GeneExpressionDataset
+
+available_datasets = {
+    "cbmc": "CBMC_8K_13AB",
+    "pbmc": "PBMC_vs_flow"
+}
+
+
+class CiteSeqDataset(GeneExpressionDataset):
+    def __init__(self, name='cbmc', save_path='data/citeSeq/'):
+        self.save_path = save_path + name + '/'
```

Comment:
`self.save_path` should equal `save_path`. Maybe call it `save_path` something else if we can't just store it directly under `save_path`.